components/list/bullet/README.md

```diff
@@ -0,0 +1,38 @@
+# ListBullet
+
+> The status spot can has the success, error and alert color.
+> We can use the status: high, medium or low to assign the different colors.
+
+<!-- ![](./assets/preview.png) -->
+
+## Installation
+
+```sh
+$ npm install @schibstedspain/sui-list-bullet --save
+```
+
+## Usage
+
+### Basic usage
+```js
+import ListBullet from '@schibstedspain/sui-list-bullet'
+
+const listItems = [
```

Comment:
check out this recommendation https://github.schibsted.io/scmspain/frontend-all-wiki/wiki/How-To-Pass-Wrapped-Nodes-As-Props

Reply:
La verdad es que está muy bien explicado. Una de las dudas que tenía de cuando usar los `componentes compuestos`

Reply:
this array is just information to fill the component tags. to be honest, i don't really understand how to match this with "passing wrapped nodes". 
🤷‍♂️

Reply:
answering my own 🔝 question, would be: creating nodes to iterate them using the array's information.
^ _ ^
@jcger component refactored. take a look!

Reply:
you would need to update this example